src/open_prime_rando/echoes/schema.json

```diff
@@ -97,6 +97,43 @@
             "required": [
                 "suits"
             ]
+        },
+        "tweaks": {
+            "type": "object",
+            "description": "Allows arbitrary changes to the tweaks",
+            "propertyNames": {
+                "enum": [
+                    "TweakGui",
+                    "TweakTargeting",
+                    "TweakPlayerRes",
+                    "TweakPlayerControls2",
+                    "TweakParticle",
+                    "TweakGuiColors",
+                    "TweakGame",
+                    "TweakPlayer2",
+                    "TweakSlideShow",
+                    "TweakBall",
+                    "TweakAutoMapper",
+                    "TweakPlayerControls",
+                    "TweakPlayerGunMuli",
+                    "TweakPlayerGun",
+                    "TweakCameraBob",
+                    "TweakPlayer"
+                ]
+            },
+            "additionalProperties": {
+                "type": "object",
+                "description": "Mapping of full property path to new value. For nested properties, include parent property names split with .",
```

Comment:
what about unknowns/unnamed properties? maybe include an example of one

Reply:
Currently needs to use the `unknown_0x2e997e0b` style, but it does remind me that if we add a proper name for these in RDS they'll break. I guess that's fine?

Reply:
yeah that's an existing issue elsewhere anyway. an example using that style should be plenty then